swmm-toolkit/src/swmm/toolkit/solver_docs.i

```diff
@@ -664,6 +664,24 @@ flowrate: double
 ) swmm_setNodeInflow;
 
 
+%feature("autodoc", 
+"Set node inflow pollutant concentration
```

Comment:
This doesn't set the inflow concentration. It sets the current node concentration. We should modify the description in line 668.